ericberringer/currency-kidz-react
# Ck learn structure

Type of change:

Testing Instructions

/learn will take you to the learn page, coins should enlarge on hover, bills won't yet, basic accordion with info.

git fetch --all 
git checkout CK-LearnStructure

Checklist:

X(I have preformed a self review)

X(Code follows style guidelines of project)

(I have commented my code. Especially w/ hard to understand code)

X(Changes I have made generate no new warnings or errors)

X(I have added test instructions that prove my fix is effective or that my feature works)

(I am not ruminating about the things my therapist told me to forget about b/c I cannot do anything about them. For example, the state of the world. )

- Fixes #
